src/app/services/auth.services.ts

```
@@ -0,0 +1,156 @@
+    const date = new Date();
+    // expires in 2 days
+    date.setTime(date.getTime() + 2 * 24 * 60 * 60 * 1000);
+
+    // Set it
+    document.cookie =
+      'Authorization' +
+      '=' +
+      tokenCookie +
+      '; expires=' +
+      date.toUTCString() +
+      '; path=/';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)